action.yml

```diff
@@ -73,15 +73,15 @@ runs:
     - name: Set Action Version
       shell: bash
       run: |
-        echo "version=2.2.0" >> $GITHUB_ENV
+        echo "version=2.2.1" >> $GITHUB_ENV
     - name: Check for reruns
       uses: pat-s/always-upload-cache@v2
       if: (! inputs.dry_run)
       with:
-        key: rainforest-run-${{ github.run_number }}-${{ github.run_attempt }}
+        key: rainforest-run-${{ github.run_id }}-${{ github.action }}-${{ github.run_attempt }}
         path: .rainforest_run_id
         restore-keys: |
-          rainforest-run-${{ github.run_number }}-
+          rainforest-run-${{ github.run_id }}-${{ github.action }}-
```

Comment:
This could potentially break reruns for workflows triggered before this is released and rerun after it is released? I couldn't [find an answer](https://docs.github.com/en/actions/managing-workflow-runs/re-running-workflows-and-jobs) as to whether workflow reruns use the exact same actions, or whether actions may be upgraded across attempts.

If it's the latter, this would only affect workflows using `rainforestapp/github-action@2` or `rainforestapp/github-action@master`